cypress/integration/team/create_a_team_spec.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Teams Suite', () => {
+    before(() => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+    });
+
+    it('Create a team', () => {
+
+        //  Click hamburger main menu
+        cy.get('#sidebarHeaderDropdownButton').click();
+
+        // dropdown menu should be visible then click create team
+        cy.get('#sidebarDropdownMenu').should('be.visible').within(function($el) {
+           cy.get('#createTeam').should('be.visible').click();
+        });
+
+        // input team name as Team Test
+        cy.get("input[type=text]").should('be.visible').type("Team Test");
```

Comment:
Instead of this selector, you may add element ID like `teamNameInput` at [display_name.jsx](https://github.com/mattermost/mattermost-webapp/blob/9d5575492f0a46b2362737c40304261cb6a282e3/components/create_team/components/display_name.jsx#L102).